dts/bindings/sensor/raspberrrypi,pico-temp.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: RaspberryPi Pico family temperature sensor node
+
+compatible: "raspberrypi,pico-temp"
+
+include: sensor-device.yaml
+
+properties:
+  io-channels:
+    required: true
+    description: ADC channel for CPU temperature sensor
+
+  vbe:
+    type: int
+    default: 706000
+    description: |
+      Vbe voltage in microvolts at 27 degree C.
+      The temperature sensor measures the Vbe voltage of a biased bipolar diode.
+
+  vbe-slope:
+    type: int
+    default: -1721
```

Comment:
Please justify the default value in the description

Reply:
Add short description for clarify it is a spec.